src/components/RegionSettings.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { MouseEvent } from "react";
+import { v4 as uuidv4 } from "uuid";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import RadioButtonCheckedIcon from "@material-ui/icons/RadioButtonChecked";
+import RadioButtonUncheckedIcon from "@material-ui/icons/RadioButtonUnchecked";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import ContinuousSlider from "./Slider";
+
+export default function RegionSettings() {
+  const [regions, setRegions] = React.useState<
+    Array<{
+      id: string;
+      type: string;
+      transparency: number;
+      activated: boolean;
```

Comment:
Według mnie lepiej od razu wynieść to do interfejsu ponieważ ten typ na pewno przyda się jeszcze w kilku miejscach